poms/bom-test/pom.xml

```diff
@@ -399,6 +403,11 @@
                 <version>${unboundid-ldapsdk.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>com.ibm.mq</groupId>
+                <artifactId>com.ibm.mq.jakarta.client</artifactId>
+                <version>${ibm.mq.client.version}</version>
```

Comment:
I think it should be enough to managed in `bom`.

Reply:
that didn't work for me as the `bom-test` does not inherit from the main `bom`

Reply:
ok I see it now, the tests actually use the `bom` through `camel-quarkus-build-parent-it`. My issue is, when I add a dependency to the `poms/bom/pom.xml` [here](https://github.com/avano/camel-quarkus/blob/ibm-mq/poms/bom/pom.xml#L6058-L6062), after I build the bom with `mvn clean install`, the artifact is not present in the installed bom in the local repository. Am I doing it wrong?

Reply:
Good point! We are flattening the BOM based on the set of artifacts that are actually required by Camel Quarkus. The set is defined by all our extensions plus some artifacts on top of it. Because the IBM MQ connector is not a transitive dep of any extensions, it needs to be added manually around here: https://github.com/apache/camel-quarkus/blob/main/poms/bom/pom.xml#L6739C30-L6750

Reply:
that did the trick, thanks!